pom.xml

```diff
@@ -103,6 +104,21 @@
 			<version>3.12.4</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>io.github.resilience4j</groupId>
+			<artifactId>resilience4j-all</artifactId>
+			<version>${resilience4j.version}</version>
+		</dependency>
```

Comment:
If you only need to introduce the modules you want to use, should we remove `resilience4j-all`?

Reply:
+1